src/components/modes/teacher/Settings.js

```diff
@@ -301,6 +352,20 @@ class Settings extends Component {
           </InputLabel>
           {selectControl}
         </FormControl>
+        <FormControl className={classes.formControlSpace}>
+          <Tooltip
+            title={
+              "Select the 'public' option to make actions compatible with learning analytics applications."
+            }
+            placement="top-start"
+            classes={{ tooltip: classes.noMaxWidth }}
```

Comment:
Why do we need this?

Reply:
To change the starting position of the text, and for the width to make the sentence to be written on one line

![image](https://user-images.githubusercontent.com/45897168/88784509-ce06ae80-d198-11ea-9f21-2ebf9969f963.png)

Reply:
OK. It doesn't disappear from the screens on small screens?

Reply:
It will usually re-adjust its position depending on size of the screen, unless the screen is very tiny. Example:
![image](https://user-images.githubusercontent.com/45897168/88787337-62bedb80-d19c-11ea-806d-5076bd936e15.png)

Reply:
OK. Sounds good.